mesonbuild/modules/python.py

```diff
@@ -238,6 +278,7 @@ def __init__(self, interpreter, python):
         self.purelib_install_path = os.path.join(prefix, install_paths['purelib'][1:])
         self.version = run_command(python, "import sysconfig; print (sysconfig.get_python_version())")
         self.platform = run_command(python, "import sysconfig; print (sysconfig.get_platform())")
+        self.is_pypy = json.loads(run_command (python, IS_PYPY_COMMAND))
```

Comment:
__[Flake8]__

[E211] whitespace before '('


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
That's fixed, not sure why it doesn't show as outdated :)